.github/workflows/freertos_plus_demos.yml

```diff
@@ -446,10 +446,20 @@ jobs:
         id: generate-credentials
         uses: FreeRTOS/CI-CD-GitHub-Actions/ssl-credential-creator@main
 
+      - name: Query device IP address
+        id: get-ip
+        run: |
+          # Query the device's IP address
+          $SERVER_IP = (Get-NetIPAddress -AddressFamily IPv4 | Where-Object { $_.IPAddress -notmatch '127.0.0.1' } | Select-Object -First 1 -ExpandProperty IPAddress)
+          Write-Host "Device IP address: $SERVER_IP"
+          echo "server_ip=$SERVER_IP" >> $env:GITHUB_ENV
+        shell: pwsh
+
       - name: Start localhost MQTT broker
         id: mqtt-broker
-        uses: FreeRTOS/CI-CD-GitHub-Actions/localhost-mqtt-broker@main
+        uses: tony-josi-aws/CI-CD-GitHub-Actions/localhost-mqtt-broker@main
```

Comment:
`tony-josi-aws/CI-CD-GitHub-Actions/localhost-mqtt-broker@main` is used for debug